### PR TITLE
A correction to the print format in g2print.F

### PR DESCRIPTION
**Pull request by kkeene44**
g2print.F (found in ungrib/src) is modified to fix the print format.

### CODE REVIEW

File: ungrib/src/g2print.F [hunk 1]
```diff
@@ -488,10 +488,10 @@ SUBROUTINE r_grib2(junit, gribflnm, hdate,  &
...
 
           if (debug_level .le. 50) then
-          write(6,*) '---------------------------------------------------------------------------------------'
-          write(6,*) ' rec Prod Cat Param  Lvl    Lvl      Lvl     Prod    Name            Time          Fcst'
-          write(6,*) ' num Disc     num    code   one      two     Templ                                 hour'
-          write(6,*) '---------------------------------------------------------------------------------------'
+          write(6,'(a87)') '---------------------------------------------------------------------------------------'
+          write(6,'(a87)') ' rec Prod Cat Param  Lvl    Lvl      Lvl     Prod    Name            Time          Fcst'
+          write(6,'(a87)') ' num Disc     num    code   one      two     Templ                                 hour'
+          write(6,'(a87)') '---------------------------------------------------------------------------------------'
```
Comment (mgduda) [inline ungrib/src/g2print.F:494]: Would it be simpler to omit the width and use a format of `'(a)'`?

**Comment by weiwangncar** [comment]
@mgduda I tested your suggestion on Cheyenne using ifort, and it does work. I'd agree that if 'a' is more general and works everywhere, we should go with 'a'.

**Comment by mgduda** [comment]
@weiwangncar Since we don't "squash-and-merge" in the WPS repository, could you rebase this PR so that there's just a single commit (that uses `'(a)'` format)? That would help to keep the commit history tidy.

**Comment by weiwangncar** [comment]
@kkeene44 Can you help with this 'rebasing' thing?

**Comment by kkeene44** [comment]
@weiwangncar 
Yes. I'm swamped for the rest of the day, but I'll get to it ASAP in the next couple of days.

**Comment by weiwangncar** [comment]
@kkeene44 That is fine. Thanks.
@mgduda Is there a way to do this on Github itself?

**Comment by mgduda** [comment]
> @kkeene44 That is fine. Thanks. @mgduda Is there a way to do this on Github itself?

Not that I know of.

**Comment by kkeene44** [comment]
@mgduda @weiwangncar 
Okay, I think I rebased correctly. Hopefully.

**Comment by weiwangncar** [comment]
@kkeene44 Thanks. Is it possible to change the final commit message? If so, can we change it to 'Fix a print format'?

**Comment by mgduda** [comment]
> @kkeene44 Thanks. Is it possible to change the final commit message? If so, can we change it to 'Fix a print format'?

I think the commit message that @kkeene44 used is more descriptive. "Fix a print format" doesn't really explain what was wrong with the existing format, and it doesn't mention which part of the WPS is affected.

**Comment by weiwangncar** [comment]
@kkeene44 @mgduda Except the starting code has the format write(6,*), not write(6,'a87')...

**Comment by kkeene44** [comment]
@mgduda @weiwangncar 
The commit message has been modified to correct the descriptive message.

**Review by mgduda — APPROVED** [review]

**Comment by mgduda** [comment]
@weiwangncar Does the updated commit message look alright to you?

**Comment by weiwangncar** [comment]
@mgduda I'm ok with the change. Thanks.

**Pull request merged**